coverage/html/common/database.dart.func-sort-c.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
Same thing as @anlhu mentioned earlier, `amber.png` probably shouldn't be pushed, and the coverage files seem to be duplicated as well?